packages/ubuntu_wizard/lib/src/l10n/ubuntu_localizations.dart

```diff
@@ -0,0 +1,223 @@
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:intl/intl.dart' as intl;
+
+import 'ubuntu_localizations_en.dart';
+import 'ubuntu_localizations_es.dart';
+import 'ubuntu_localizations_fr.dart';
+import 'ubuntu_localizations_it.dart';
+import 'ubuntu_localizations_nl.dart';
+import 'ubuntu_localizations_oc.dart';
+import 'ubuntu_localizations_pt.dart';
+import 'ubuntu_localizations_ru.dart';
+
+/// Callers can lookup localized strings with an instance of UbuntuLocalizations returned
+/// by `UbuntuLocalizations.of(context)`.
+///
+/// Applications need to include `UbuntuLocalizations.delegate()` in their app's
+/// localizationDelegates list, and the locales they support in the app's
+/// supportedLocales list. For example:
+///
+/// ```
+/// import 'l10n/ubuntu_localizations.dart';
+///
+/// return MaterialApp(
+///   localizationsDelegates: UbuntuLocalizations.localizationsDelegates,
+///   supportedLocales: UbuntuLocalizations.supportedLocales,
+///   home: MyApplicationHome(),
+/// );
+/// ```
+///
+/// ## Update pubspec.yaml
+///
+/// Please make sure to update your pubspec.yaml to include the following
+/// packages:
+///
+/// ```
+/// dependencies:
+///   # Internationalization support.
+///   flutter_localizations:
+///     sdk: flutter
+///   intl: any # Use the pinned version from flutter_localizations
+///
+///   # rest of dependencies
+/// ```
+///
+/// ## iOS Applications
```

Comment:
Is this copy-pasted from somewhere?

Reply:
auto-generated by `flutter gen-l10n`: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/localizations/gen_l10n_templates.dart#L54